xml/sle_update_online.xml

```diff
@@ -509,10 +511,9 @@ Continue? [y/n/? shows all options] (y):</screen>
   <important>
    <title>Installation sources</title>
    <para>
-    This migration path requires you to provide the installation sources for
-    the new service pack in a place that can be accessed by the machine you are
-    going to migrate. This can for example be achieved by setting up an &rmt;
-    server or an SLP server.
+   This migration path requires that the system you are going to migrate has
+   access to the installation sources. For example, this can done by setting up
+   an &rmt; server or an SLP server.
```

Comment:
Indention off by -1
```suggestion
    This migration path requires that the system you are going to migrate has
    access to the installation sources. For example, this can done by setting up
    an &rmt; server or an SLP server.
```